packages/server/src/services/agent.service.ts

```
@@ -110,7 +107,7 @@ export class AgentService implements IAgentService {
   }
 
   async *handleUserRequestWebsocket(
-    agent: AgentSystem | any,
+    agent: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)